core/metrics-core-impl/src/main/java/org/hawkular/metrics/core/impl/MetricsServiceImpl.java

```diff
@@ -395,15 +395,20 @@ public void setDateTimeService(DateTimeService dateTimeService) {
         return dataAccess.findAllTenantIds()
                 .flatMap(Observable::from)
                 .map(row -> row.getString(0))
-                .flatMap(dataAccess::findTenant)
-                .flatMap(Observable::from)
-                .map(Functions::getTenant);
+                .distinct()
```

Comment:
Is this ```distinct()``` call doing anything useful? Isn't is operating on a com.datastax.driver.core.Row object? I believe com.datastax.driver.core.ArrayBackedRow is the only Row implementation, and it does not implement ```equals`` in which case every row object would be considered distinct, right?